<!DOCTYPE html>
<html>

  <head>
    <meta charset='utf-8'>
    <meta http-equiv="X-UA-Compatible" content="chrome=1">
    <meta name="description" content="Breast Functional Genomics (BFG) : Data for &quot;The Functional Genomic Landscape  of Human Breast Cancer Drivers, Vulnerabilities, and Resistance&quot;">

    <link rel="stylesheet" type="text/css" media="screen" href="stylesheets/stylesheet.css">

    <title>Breast Functional Genomics (BFG)</title>
  </head>

  <body>

    <!-- HEADER -->
    <div id="header_wrap" class="outer">
        <header class="inner">
          <a id="forkme_banner" href="https://github.com/neellab/bfg">View on GitHub</a>

          <h1 id="project_title">Breast Functional Genomics (BFG)</h1>
          <h2 id="project_tagline">Data for &quot;The Functional Genomic Landscape  of Human Breast Cancer Drivers, Vulnerabilities, and Resistance&quot;</h2>

            <section id="downloads">
              <a class="zip_download_link" href="https://github.com/neellab/bfg/zipball/master">Download this project as a .zip file</a>
              <a class="tar_download_link" href="https://github.com/neellab/bfg/tarball/master">Download this project as a tar.gz file</a>
            </section>
        </header>
    </div>

    <!-- MAIN CONTENT -->
    <div id="main_content_wrap" class="outer">
      <section id="main_content" class="inner">
        <h2>
<a id="manuscript" class="anchor" href="#manuscript" aria-hidden="true"><span class="octicon octicon-link"></span></a>Manuscript</h2>

<p>"Functional Genomic Landscape  of Human Breast Cancer Drivers,  Vulnerabilities, and Resistance"</p>

<p>Richard Marcotte*, Azin Sayad*, Kevin R. Brown, Felix Sanchez-Garcia, Jüri Reimand, Maliha Haider, Carl Virtanen, James E. Bradner, Gary D. Bader, Gordon B. Mills, Dana Pe’er, Jason Moffat and Benjamin G. Neel</p>

<p>(* Equal Contributions)</p>

<p>PMID:</p>

<p>[LINK] [PDF]</p>

<h2>
<a id="links-to-data-on-geo" class="anchor" href="#links-to-data-on-geo" aria-hidden="true"><span class="octicon octicon-link"></span></a>Links to Data on GEO</h2>

<p>shRNA screens (<a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE74702%5D">GSE74702</a>)</p>

<p>RNAseq (<a href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE73526">GSE73526</a>)</p>

<h2>
<a id="functional-genomics-and-proteomics-data-files" class="anchor" href="#functional-genomics-and-proteomics-data-files" aria-hidden="true"><span class="octicon octicon-link"></span></a>Functional, Genomics and Proteomics Data Files</h2>

<p><strong>shRNA screens</strong></p>

<ul>
<li>
<p><a href="http://shrna">ExpressionSet</a>: RData format file containing a Bioconductor ExpressionSet object combining</p>

<ul>
<li>Per-sample (cell line + time-point + replicate), per-hairpin log2 measurements matrix, accessed using the <code>exprs()</code> R function.</li>
<li>Per-hairpin, per-gene annotations data frame, with the same number of rows as the measurement matrix.</li>
<li>Per sample annotation data frame, including information such as cell line and time-point. This number of rows in this data frame matches the number of columns in the measurements matrix.</li>
</ul>
</li>
<li><p><a href="https://github.com/neellab/bfg/blob/gh-pages/data/rnaseq/breast_rnaseq_qn.txt.zip?raw=true">per-gene normalized log2 FPKM</a>: As above, but also Quantile-Normalized, and a floor value set at log2(0.1) FPKM . This version was used in the manuscript analyses.</p></li>
</ul>

<p><strong>RNASeq</strong></p>

<p>RNAseq Per-Gene log2-FPKM</p>

<ul>
<li><a href="http://RNASEQ">per-gene log2 FPKM</a></li>
<li>
<a href="https://github.com/neellab/bfg/blob/gh-pages/data/rnaseq/breast_rnaseq_qn.txt.zip?raw=true">per-gene normalized log2 FPKM</a>: As above, but also Quantile-Normalized, and a floor value set at log2(0.1) FPKM . This version was used in the manuscript analyses.</li>
</ul>

<p><strong>CNA</strong></p>

<p><a href="">CNA Per-Gene Log-R Ratios</a></p>

<p><strong>miRNA</strong></p>

<p>miRNA [PROCESSED] [BATCH1] [BATCH2] [BATCH3]</p>

<p><strong>RPPA</strong></p>

<p><a href="https://github.com/neellab/bfg/blob/gh-pages/data/rppa/cell_line_rppa.xlsx?raw=true">RPPA</a></p>

<h2>
<a id="annotation-files" class="anchor" href="#annotation-files" aria-hidden="true"><span class="octicon octicon-link"></span></a>Annotation Files</h2>

<p><a href="https://github.com/neellab/bfg/blob/gh-pages/data/annotations/cell_line_subtypes.txt.zip?raw=true">Cell line subtypes</a>: sub-typing of each cell line according to</p>

<ul>
<li>receptor expression and status (AR, ERBB2, ESR1, PGR)</li>
<li>three-receptor classification (ER/ERBB2/TNBC)</li>
<li>Curtis Integrative Subtypes (IntClust 1 through 10)</li>
<li>Intrinsic classification (Luminal A, Luminal B, Her2, Basal, Normal-like) with Claudin Low status.</li>
<li>Lehmann TNBC subtypes (Basal Like 1, Basal Like 2, Immunomodulatory, Luminal Androgen Receptor, Mesenchymal, Mesenchymal Stem-Like)</li>
<li>Neve classification (luminal, basal A, basal B) with luminal cell lines further classified as her2 added based on high ERBB2 expression status.</li>
<li>Unsupervised clustering of RNASeq, RPPA and shRNA screen data.</li>
</ul>

<p><a href="https://github.com/neellab/bfg/blob/gh-pages/data/annotations/hairpin_annotations.txt.zip?raw=true">updated shRNA annotations</a></p>
      </section>
    </div>

    <!-- FOOTER  -->
    <div id="footer_wrap" class="outer">
      <footer class="inner">
        <p class="copyright">Breast Functional Genomics (BFG) maintained by <a href="https://github.com/neellab">neellab</a></p>
        <p>Published with <a href="https://pages.github.com">GitHub Pages</a></p>
      </footer>
    </div>

    

  </body>
</html>
